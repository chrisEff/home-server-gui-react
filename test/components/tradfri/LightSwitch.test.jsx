const React = require('react')
const LightSwitch = require('../../../src/components/tradfri/LightSwitch')

const {shallow} = require('enzyme')

describe('LightSwitch', () => {
	it('should render correctly', () => {
		const wrapper = shallow(<LightSwitch state={1} />)
		expect(wrapper).toMatchSnapshot()
	})
})
