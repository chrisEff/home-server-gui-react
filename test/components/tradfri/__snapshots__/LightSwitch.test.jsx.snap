// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LightSwitch should render correctly 1`] = `
<div
  className="bulb"
  onClick={[Function]}
>
  ðŸ’¡
</div>
`;
